devicetypes/keen-home/keen-home-smart-vent.src/keen-home-smart-vent.groovy

```diff
@@ -327,35 +311,79 @@ private def makeSerialResult(serial) {
         value: serial,
         descriptionText: "${linkText} has serial ${serial}" ]
 }
-/**** COMMAND METHODS ****/
-// def mfgCode() {
-// 	 ["zcl mfg-code 0x115B", "delay 200"]
-// }
 
+// takes a level from 0 to 100 and translates it to a ZigBee move to level with on/off command
+private def makeLevelCommand(level) {
```

Comment:
On our device, a (ZigBee) level of 0xFF is a reserved value, which I don't think the `zigbee.setLevel` method would be able to take into account. Also, I don't know what command `setLevel` sends, but for our device it's important the command is a "move to level with on/off" rather than a "move to level".